pipelines/internal/update-fbc-catalog/update-fbc-catalog.yaml

```diff
@@ -60,6 +60,7 @@ spec:
       description: The revision in the taskGitUrl repo to be used
   tasks:
     - name: update-fbc-catalog-task
+      retries: 3
```

Comment:
Hmm, I thought we were aiming for adding retries to managed pipelines only. But it's true the acceptance criteria don't specify it, it just says "all pipelines". And since, as you said, the simple signing pipeline does have this also, I guess it should be ok. @theflockers do you see any risk in adding retries here?

Reply:
I removed it since this task only add "retries" to managed pipelines.